packages/nextjs/src/app-beta/ClerkProvider.tsx

```diff
@@ -11,6 +17,10 @@ type NextAppClerkProviderProps = {
 } & Omit<IsomorphicClerkOptions, keyof PublishableKeyOrFrontendApi> &
   Partial<PublishableKeyOrFrontendApi>;
 
+/**
+ * @deprecated Use imports from `@clerk/nextjs` instead.
+ * For more details, consult the middleware documentation: https://clerk.com/docs/nextjs/middleware'
```

Comment:
Nit: In all JSDoc comments you have a `'` at the end of the URL

Reply:
nice catch! I removed it.